go-kit/metrics/testmetrics/provider.go

```diff
@@ -170,6 +170,24 @@ func (p *Provider) CheckNoCounter(name string, labelValues ...string) {
 	}
 }
 
+// CheckCounterExists checks that there is registered counter with the name
+// provided and value is greater than 0.
```

Comment:
Technically, your function checks that the value _is not zero_ which is different from _greater than 0_. Let's fix the logic or update the comment to match.